days/day-04/output.txt

```diff
@@ -1 +1,2 @@
-<insert output>
+Part 1 => 202
```

Comment:
output becomes fasit for all solutions, so the convention is to only write the result:

202
137